virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -231,4 +231,7 @@ Legacy OpenSSL clients that do not support EMS or TLS 1.3 now cannot connect to
 ==== Web console
 //CNV-38293: 4.16 - Ronen Sde-Or said this was still an issue
 * When you first deploy an {product-title} cluster, creating VMs from templates or instance types by using the web console, fails if you do not have `cluster-admin` permissions.
```

Comment:
```suggestion
* When you deployed an {product-title} cluster, and then created VMs from templates or instance types by using the web console, the VMs failed if you did not have `cluster-admin` permissions. As a workaround, the cluster administrator must first xref:../../nodes/pods/nodes-pods-configmaps.adoc#nodes-pods-configmap-create-from-console_configmaps[create a config map] to provide users access to use templates and instance types to create VMs. (link:https://issues.redhat.com/browse/CNV-38284[*CNV-38284*])
```

Reply:
This is a 4.15 release note that has been carried over. I only fixed the link syntax.